packages/third-parties/src/google/gtm.tsx

```diff
@@ -8,7 +8,7 @@ import type { GTMParams } from '../types/google'
 let currDataLayerName: string | undefined = undefined
 
 export function GoogleTagManager(props: GTMParams) {
```

Comment:
Also include `trackingHost` in GTMParams type

Reply:
thanks for the feedback! updated